blob/service.go

```diff
@@ -197,12 +195,12 @@ func (s *Service) getByCommitment(
 
 	namespacedShares, err := s.shareGetter.GetSharesByNamespace(ctx, header.DAH, nID)
 	if err != nil {
-		if errors.Is(err, share.ErrNotFound) {
+		if errors.Is(err, share.ErrNamespaceNotFound) ||
+			errors.Is(err, share.ErrNotFound) {
```

Comment:
Does it return share.ErrNotFound?

Reply:
IIRC, yes. @distractedm1nd could you please confirm?

Reply:
Yes, for example in the case of a context deadline (which is a failure to find the data, hence ErrNotFound)